Gemfile

```diff
@@ -1,6 +1,6 @@
-ruby "2.2.4"
 source 'https://rubygems.org'
 
+ruby File.read('.ruby-version').strip
```

Comment:
It's a way of keeping them both in sync, so would stop things like this https://circleci.com/gh/withassociates/with_links-2014/180